gradle/libs.versions.toml

```diff
@@ -1,30 +1,28 @@
 [versions]
-androidGradlePlugin = "8.4.1"
+androidGradlePlugin = "8.5.1"
 kotlin = "2.0.0"
 ktlint = "12.1.0"
 detekt = "1.23.4"
-roborazzi = "1.15.0"
+roborazzi = "1.22.2"
 dokka = "1.9.20"
 binaryValidator = "0.15.0-Beta.2"
 publishToS3 = "0.10.0"
 openapi = "7.4.0"
 retrofit = "2.11.0"
 okhttp = "4.12.0"
-ktx = "1.12.0"
+ktx = "1.13.1"
 desugarJdk = "2.0.4"
 junit = "4.13.2"
 mockk = "1.13.9"
-espresso = "3.5.1"
-androidXJunit = "1.1.5"
-appcompat = "1.6.1"
-material = "1.11.0"
+appcompat = "1.7.0"
+material = "1.12.0"
 ucrop = "2.2.8"
```

Comment:
```suggestion
ucrop = "2.2.9"
```

Reply:
Yeah, I've noticed AS doesn't always suggest the latest available version 🤔 not sure why it lags but it didn't highlight `ucrop` at all for me.

Reply:
Yeah, I don't know why that happens sometimes. In this case, it was highlighted for me.